src/libs/CollectionUtils.ts

```diff
@@ -0,0 +1,10 @@
+function lastItem<T>(object: Record<string, T> = {}): T {
```

Comment:
This might have changed since the function was first written, but in the comment of the function, it's expecting an object with numbers for keys. Is this still it's current usage today?